src/nomad_analysis/jupyter/schema.py

```diff
@@ -293,8 +219,10 @@ def process_query_for_inputs(
         entries = []
 
         # extend the entries with the data from query_for_inputs
-        if self.query_for_inputs is not None:
-            entries.extend(self.query_for_inputs['data'])
+        if self.query_for_inputs:
+            for query in self.query_for_inputs:
+                if query.get('data') is not None:
+                    entries.extend(query['data'])
```

Comment:
This looks a little bit awkward. Maybe something like:
```suggestion
        # extend the entries with the data from query_for_inputs
        if self.query_for_inputs:
            entries = [query['data'] for query in self.query_for_inputs if 'data' in query]
```